style.css

```diff
@@ -23,48 +25,129 @@ a {
   text-decoration: none;
 }
 
-header {
+/* header {
   position: fixed;
   top: 0%;
   z-index: 5;
   box-shadow: 0.1rem 0.1rem rgba(245, 243, 243, 0.918);
   background: #fff;
-  overflow: hidden;
-  background-color: #fff;
+} */
+
+.navbar {
+  min-height: 70px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 24px;
+}
+
+.right-nav {
+  color: black;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 60px;
+  position: fixed;
+  left: -100%;
+  top: 70px;
+  flex-direction: column;
+  background-color: #5ac057;
   width: 100%;
-  height: 916px;
+  text-align: center;
+  transition: 0.3s;
 }
```

Comment:
Your mobile and desktop versions are still very dissimilar to the template design. Please fix them.